src/background/Background.ts

```diff
@@ -3,23 +3,31 @@ import { runtime, tabs } from 'webextension-polyfill'
 import { PaymentFlowService } from '@/background/grantFlow'
 
 import getSendingPaymentPointerHandler from '../messageHandlers/getSendingPaymentPointerHandler'
+import getStorageData from '../messageHandlers/getStorageData'
 import isMonetizationReadyHandler from '../messageHandlers/isMonetizationReadyHandler'
 import setIncomingPointerHandler from '../messageHandlers/setIncomingPointerHandler'
+import { defaultData } from './StorageService'
 import { tabChangeHandler, tabUpdateHandler } from './tabHandlers'
 
 class Background {
   private messageHandlers: any = [
     isMonetizationReadyHandler,
     setIncomingPointerHandler,
     getSendingPaymentPointerHandler,
+    getStorageData,
   ]
   private subscriptions: any = []
   // TO DO: remove these from background into storage or state & use injection
   grantFlow: PaymentFlowService | null = null
   spentAmount: number = 0
   paymentStarted = false
 
-  constructor() {}
+  constructor({ storageService }: any) {
+    storageService
```

Comment:
initialisation of data should be done when create a new instance of storage, because it's a singleton